core/src/test/resources/org/elasticsearch/action/fieldstats/fieldstats-index-constraints-request.json

```diff
@@ -28,6 +28,16 @@
       "max_value" : {
         "lt": 9
       }
+    },
+    "field1": {
```

Comment:
the other fields are already without a format, so the parsing is properly logic is properly tested.